deepcell/model_zoo/panopticnet.py

```diff
@@ -366,11 +394,10 @@ def PanopticNet(backbone,
     features = [pyramid_dict[key] for key in pyramid_levels]
 
     if frames_per_batch > 1:
-        if temporal_mode in ['conv', 'lstm', 'gru']:
-            temporal_features = [__merge_temporal_features(
-                feature, mode=temporal_mode) for feature in features]
-            for f, k in zip(temporal_features, pyramid_dict.keys()):
-                pyramid_dict[k] = f
+        temporal_features = [__merge_temporal_features(
+            feature, mode=temporal_mode) for feature in features]
+        for f, k in zip(temporal_features, pyramid_dict.keys()):

```

Comment:
Does this give us the expected answer if `temporal_mode` is `None` (the default value)?